src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -228,6 +229,20 @@ public virtual int MaximumRedirection
         }
         private int _maximumRedirection = -1;
 
+        /// <summary>
+        /// Gets or sets the RetryCount property, which determines the number of retries of a failed web request.
+        /// </summary>
+        [Parameter]
+        [ValidateRange(0, Int32.MaxValue)]
+        public virtual int RetryCount { get; set; } = 0;
+
+        /// <summary>
+        /// Gets or sets the RetryIntervalSec property, which determines the number seconds between retries.
+        /// </summary>
+        [Parameter]
+        [ValidateRange(1, Int32.MaxValue)]
+        public virtual int RetryIntervalSec { get; set; } = 60;
```

Comment:
60 seconds as the default just feels too long. So many times testing this feature and forgetting the `-RetryIntervalSec` I felt like the console froze or the thread locked. between 10 and 15 seconds I can't decide which is better. 10 feels more natural, but 15 is probably better. anything over 15 really starts to feel like the command has frozen.

Reply:
According to Azure docs [here](https://docs.microsoft.com/en-us/aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/transient-fault-handling#use-smart-retryback-off-logic-to-mitigate-the-effect-of-transient-failures) the sample shows 5 seconds. @markekraus  @SteveL-MSFT what do you think?

Reply:
I would agree with 5 seconds for transient errors, but we are covering more than transient errors (for example, 404). But... maybe it's ok. 5 seconds is still only 12 requests a minute which isn't that spammy, IMO.

Reply:
Changing it to 5 seconds, if we get feedback about it to increase the defaults, we can reconsider.